CONTRIBUTING.md

```diff
@@ -100,7 +100,25 @@ specify exactly the dependency you want to update, eg.
 
 ## Protobuf
 
-When working with [protobuf](https://developers.google.com/protocol-buffers) there are a few things you should know. We use [buf](https://buf.build/) for our linting and breaking changes checking. If you would like to run linting and check if the changes you have made are breaking then you will have to install the needed dependencies with `make buf`. Then the linting cmd will be `make proto-lint` and the breaking changes check will be `make proto-check-breaking`. To generate new stubs based off of your changes you can run `make proto-gen` (you can do this outside of GOPATH).
+We use [Protocol Buffers](https://developers.google.com/protocol-buffers) along with [gogoproto](https://github.com/gogo/protobuf) to generate code for use across Tendermint Core.
+
+For linting and checking breaking changes, we use [buf](https://buf.build/). If you would like to run linting and check if the changes you have made are breaking then you will have to install the needed dependencies with `make buf`. Then the linting cmd will be `make proto-lint` and the breaking changes check will be `make proto-check-breaking`.
+
+To generate new stubs based off of your changes you can run `make proto-gen` after installing `protoc` and gogoproto.
+
+### Installation Instructions
+
+To install `protoc`, download an appropriate release (https://github.com/protocolbuffers/protobuf) and then move the provided binaries into your PATH (follow instructions in README included with the download).
+
+To install `gogoproto`, do the following:
+
+```sh
+$ go get github.com/gogo/protobuf/gogoproto
```

Comment:
```suggestion
$ make protobuf
```

this command will do the same work

Reply:
haha, ok. because I like to know what I'm running, my preference is to not have/use make aliases for single line commands (unless that one line is very complicated!), but it's six of one, half a dozen of the other

Reply:
ah okay, now that i look at the tools.mk file we are downloading gogoproto into the protoc dir. a bit confused what is going on there.

Reply:
Oh lord, ok.